### PR TITLE
chore(e2e): Modification for visualisation of Experiment Details

### PR DESCRIPTION
**Pull request by oumkale**
Signed-off-by: Oum Kale <oumkale@chaosnative.com>

For Demo Link : [Link](https://oumkale.github.io/litmus-e2e)

This is modifying the visualisation of Validation tests.
 - Heat Map is replaced by Bar Charts/Table Charts.
 - Changed Coverage Metrics to Experiment Details
 - Removed heat map images
 - Added logos for platforms
 - Code Refactoring
 - For more coverage details On-Clicking Bar Graph will redirect to ```.master-plan.yml```
 
 For example:

Platform tested : 

Screenshot from 2021-04-14 18-46-10

On Selecting Actual or Expected.

Screenshot from 2021-04-14 18-44-35

Application and Run Time Based Coverage : 

Screenshot from 2021-04-14 18-44-37

### CODE REVIEW

File: heatmap/kubelet-service-kill.html [hunk 1]
```diff
@@ -1,35 +1,59 @@
...
+    <div style="width: 1000px; text-align: center; margin: 0 auto;">
+      <img src="../images/platforms/GKE.png" alt="HTML5 Icon" style="width:60px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/EKS.png" alt="HTML5 Icon" style="width:140px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/AKS.svg" alt="HTML5 Icon" style="width:80px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/minikube.svg" alt="HTML5 Icon" style="width:80px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/kubeadm.png" alt="HTML5 Icon" style="width:100px;height:60px; margin-right: 5px;">
+    </div>
+    <br><br>
+    <h1><u>Kubelet Service Kill Experiment Details</u></h1>
+    <p>Kubelet Service Kill fills up ephemeral storage of an application pod. Here are the different experiment details.</p>
```
Comment (Jonsy13) [inline heatmap/kubelet-service-kill.html:24]: This description is wrong.

> Kills the kubelet service on the application node to check the resiliency.

File: heatmap/node-taint.html [hunk 2]
```diff
@@ -1,35 +1,61 @@
...
+    <div style="width: 1000px; text-align: center; margin: 0 auto;">
+      <img src="../images/platforms/GKE.png" alt="HTML5 Icon" style="width:60px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/EKS.png" alt="HTML5 Icon" style="width:140px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/AKS.svg" alt="HTML5 Icon" style="width:80px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/kubeadm.png" alt="HTML5 Icon" style="width:100px;height:60px; margin-right: 5px;">
+    </div>
+    <br><br>
+    <h1><u>Node Taint Experiment Details</u></h1>
+    <p>
+      Node Taint exhaust memory resources on the kubernetes node.
```
Comment (Jonsy13) [inline heatmap/node-taint.html:24]: Is it correct? @uditgaurav

File: heatmap/pod-network-duplication.html [hunk 3]
```diff
@@ -1,35 +1,60 @@
...
+      <img src="../images/platforms/GKE.png" alt="HTML5 Icon" style="width:60px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/EKS.png" alt="HTML5 Icon" style="width:140px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/AKS.svg" alt="HTML5 Icon" style="width:80px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/kind.png" alt="HTML5 Icon" style="width:100px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/minikube.svg" alt="HTML5 Icon" style="width:80px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/kubeadm.png" alt="HTML5 Icon" style="width:100px;height:60px; margin-right: 5px;">
+    </div>
+    <br><br>
+    <h1><u>Pod Network Duplication Experiment Details</u></h1>
+    <p>Pod Network Duplication, Here are the different experiment details.</p>
```
Comment (Jonsy13) [inline heatmap/pod-network-duplication.html:25]: The description is not there?

File: heatmap/pod-network-latency.html [hunk 4]
```diff
@@ -1,35 +1,60 @@
...
+      <img src="../images/platforms/GKE.png" alt="HTML5 Icon" style="width:60px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/EKS.png" alt="HTML5 Icon" style="width:140px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/AKS.svg" alt="HTML5 Icon" style="width:80px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/kind.png" alt="HTML5 Icon" style="width:100px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/minikube.svg" alt="HTML5 Icon" style="width:80px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/kubeadm.png" alt="HTML5 Icon" style="width:100px;height:60px; margin-right: 5px;">
+    </div>
+    <br><br>
+    <h1><u>Pod Network Latency Experiment Details</u></h1>
+    <p>Pod Network Latency network latency into application pod. Here are the different experiment details.</p>
```
Comment (Jonsy13) [inline heatmap/pod-network-latency.html:25]:
```suggestion
    <p>Pod Network Latency injects network latency into application pod. Here are the different experiment details.</p>
```
Can we have this minor change

File: heatmap/pod-network-loss.html [hunk 5]
```diff
@@ -1,35 +1,60 @@
...
+      <img src="../images/platforms/GKE.png" alt="HTML5 Icon" style="width:60px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/EKS.png" alt="HTML5 Icon" style="width:140px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/AKS.svg" alt="HTML5 Icon" style="width:80px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/kind.png" alt="HTML5 Icon" style="width:100px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/minikube.svg" alt="HTML5 Icon" style="width:80px;height:60px; margin-right: 5px;">
+      <img src="../images/platforms/kubeadm.png" alt="HTML5 Icon" style="width:100px;height:60px; margin-right: 5px;">
+    </div>
+    <br><br>
+    <h1><u>Pod Network Loss Experiment Details</u></h1>
+    <p>Pod Network Loss network loss into application pod. Here are the different experiment details.</p>
```
Comment (Jonsy13) [inline heatmap/pod-network-loss.html:25]:
```suggestion
    <p>Pod Network Loss injects network loss into application pod. Here are the different experiment details.</p>
```

**Review by Jonsy13 — CHANGES_REQUESTED** [review]
Thanks, @oumkale , PTAL some minor changes.

@uditgaurav  As now we are not showing heatmaps, the directory name and routes also needs to change?

**Comment by Jonsy13** [comment]
Also, i can see, On the bar graph, the coverage label is along x-axis, shouldn't it be along y-axis?

File: heatmap/node-taint.html (hunk 2 above)
Comment (uditgaurav) [reply heatmap/node-taint.html:24]: needs to be updated! You can refer from [experiment metadata](https://docs.litmuschaos.io/docs/pod-delete/#experiment-metadata) @oumkale

Comment (oumkale) [reply heatmap/node-taint.html:24]: Thank you @uditgaurav and @Jonsy13 for pointing out. have updated.

**Comment by oumkale** [comment]
That is not a label @Jonsy13 Just a name Test graph name. axis labels are different. We are using using charts. So graphs looks like this

**Comment by oumkale** [comment]
Hi @uditgaurav @Jonsy13,
PTAL route has been changed to exam: ``` experimentDetails/pod-delete.html ```

**Review by Jonsy13 — APPROVED** [review]

**Review by uditgaurav — APPROVED** [review]

**Pull request merged**